openspec/changes/phase-13-wave-6-pidff/tasks.md

```
@@ -0,0 +1,709 @@
+- [ ] Reuse existing axis/button emit helpers from `buildFromOutput`
+  (extract them to private `emitAxes` / `emitButtons` if not already).
+- [ ] After axis/button blocks, emit Usage Page Physical Interface
+  Device (`0x05 0x0F`) marker.
+
+### T1b: Emit all 12 PID mandatory reports
+
+- [ ] Set Effect Report (ID 1) — Effect Type usage 0x25, Duration 0x50,
+  Sample Period 0x54, Gain 0x52, Trigger Button 0x53, Axes Enable 0x55,
+  Direction X 0x56, Direction Y 0x57. Layout matches USB PID 1.01 §4.1.
+- [ ] Set Envelope Report (ID 2) — Attack Level 0x5B, Fade Level 0x5E,
+  Attack Time 0x5C, Fade Time 0x5D.
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)